global/autoload/audio/audio_queue.gd

```diff
@@ -0,0 +1,49 @@
+class_name AudioQueue extends Node
+
+@export var stream_player_count: int = 4
+@export_enum(&"Master", &"Music", &"SFX") var bus: String = &"Master"
+
+var available_stream_players: Array[AudioStreamPlayer] = []
```

Comment:
Comment for all new variables: I like to prefix variables with `_` if they are used internally (private) i.e. only inside the script/node/scene that they are defined in. (Coming from my Java habits.)

What do you think about this?

Reply:
yeah that's definitely best practice, I'll update it!